templates/server-acl-init-job.yaml

```diff
@@ -101,6 +129,8 @@ spec:
                 {{- end }}
                 {{- if .Values.connectInject.enabled }}
                 -create-inject-auth-method=true \
+                -inject-auth-method-host={{ .Values.connectInject.authMethodConfig.host }} \
```

Comment:
Yeah we should. I was hoping to get away with less yaml templating logic, but forgot that it will look weird with these flags set to empty strings when you'll try to describe the pod. I'll add the ifs. They don't need to both be set because if you haven't set them we'll just fall back to using the default (either the kube api service or the CA included with the service account token).